### PR TITLE
tpm_device: add tpm2_sign test

### PR DESCRIPTION
**Pull request by yanqzhan**
Test tpm2_sign with different hash algorithms.

### CODE REVIEW

**Comment by yanqzhan** [comment]
(1/2) type_specific.io-github-autotest-libvirt.virtual_devices.tpm_device.normal_test.tpm-crb_model.emulator.encrypted.pcrbank_default.sign_test: PASS (34.66 s)
 (2/2) type_specific.io-github-autotest-libvirt.virtual_devices.tpm_device.normal_test.tpm-crb_model.emulator.encrypted.pcrbank_default.rsaencypt_test: PASS (36.39 s)

File: libvirt/tests/src/virtual_device/tpm_device.py [hunk 1]
```diff
@@ -635,6 +645,19 @@ def test_rsaencypt_in_guest(session):
...
+        Test tpm signature in guest.
+
+        :param session: Guest session to be tested
+        """
+        prepare_rsa_key(session)
+        for hash_algorithm in ['sha1', 'sha256', 'sha384', 'sha512']:
+            status, output = session.cmd_status_output("tpm2_sign -c key.ctx -g %s -o sig.rssa msg.dat" % hash_algorithm)
+            if status:
+                test.fail("tpm2_sign failed with %s hash algorithm: %s" % (hash_algorithm, output))
+        session.cmd("rm -f primary.ctx key.pub key.priv key.ctx msg.dat")
```
Comment (dzhengfy) [inline libvirt/tests/src/virtual_device/tpm_device.py:659]: Do we need to remove `sig.rssa`?

Comment (yanqzhan) [reply libvirt/tests/src/virtual_device/tpm_device.py]: Yes, added. Thanks!

**Review by dzhengfy — APPROVED** [review]
LGTM

**Review by Yingshun — APPROVED** [review]

**Pull request merged**